doc/powered-by.md

```diff
@@ -24,33 +24,48 @@ or providing commercial products including APISIX.
 
 Users are encouraged to add themselves to this page, [issue](https://github.com/apache/apisix/issues/487) and PR are welcomed.
 
+1. <a href="https://hy.10086.cn/" rel="nofollow">中移杭研</a>
+1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
 1. <a href="https://www.360.cn/" rel="nofollow">360 奇虎</a>
+1. <a href="http://51tiangou.com/" rel="nofollow">51tiangou 大商天狗</a>
+1. <a href="http://aimiaobi.com/" rel="nofollow">aimiaobi 妙笔 AI</a>
 1. <a href="https://www.chaolian360.com/" rel="nofollow">chaolian 超链云商</a>
 1. <a href="http://www.cunw.com.cn/" rel="nofollow">cunw 湖南新云网</a>
+1. <a href="https://www.daocloud.io/" rel="nofollow">DaoCloud</a>
 1. <a href="https://www.dasouche.com/" rel="nofollow">dasouche 大搜车</a>
 1. <a href="https://www.dataoke.com/" rel="nofollow">dataoke 大淘客</a>
+1. <a href="http://dian.so/" rel="nofollow">小电科技</a>
+1. <a href="https://www.didachuxing.com/" rel="nofollow">嘀嗒出行</a>
 1. <a href="https://www.efactory-project.eu/" rel="nofollow">eFactory</a>
 1. <a href="https://www.ehomepay.com.cn/" rel="nofollow">ehomepay 理房通</a>
+1. <a href="https://ezone.work/" rel="nofollow">eZone 简单一点科技</a>
 1. <a href="https://fansup.mobi/" rel="nofollow">fansup</a>
 1. <a href="http://haieruplus.com/" rel="nofollow">haieruplus 海尔优家</a>
 1. <a href="http://www.hellowin.cn/" rel="nofollow">hellowin 好洛维</a>
 1. <a href="https://www.hellotalk.com/" rel="nofollow">HelloTalk, Inc.</a>
+1. <a href="https://www.htzhiyun.cn/" rel="nofollow">航天网信</a>
 1. <a href="http://huawei.com/" rel="nofollow">Huawei 华为</a>
+1. <a href="https://www.huya.com/" rel="nofollow">虎牙</a>
+1. <a href="https://www.ihomefnt.com/" rel="nofollow">ihomefnt 艾佳生活</a>
 1. <a href="https://www.jiandanxinli.com/" rel="nofollow">jiandanxinli 简单心理</a>
-1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
 1. <a href="https://www.ke.com/" rel="nofollow">ke.com 贝壳找房</a>
+1. <a href="https://jr.ly.com/" rel="nofollow">jr.ly 同程金服</a>
 1. <a href="https://www.meizu.com/" rel="nofollow">Meizu 魅族</a>
+1. <a href="https://www.mingyuanyun.com/" rel="nofollow">明源云客</a>
 1. <a href="https://www.jpl.nasa.gov" rel="nofollow">NASA JPL 美国国家航空航天局 喷气推进实验室</a>
-1. <a href="http://www.163.com" rel="nofollow">Netease 网易</a>
+1. <a href="https://open.youtu.qq.com/" rel="nofollow">Youtu 腾讯优图</a>
```

Comment:
Should be ordered with `Youtu`?

Reply:
I would prefer using QQ 🤔

Reply:
> I would prefer using QQ 🤔

Maybe `Tencent` could be better than `QQ` 🤣

Reply:
wow, links in this PR are ordered according to the domain 😊

Reply:
Got it 😬